tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -10,135 +10,404 @@
+                    validators=[validators.isNumber()],
+                    is_encrypted=False,
+                ),
+                Field(
+                    item="17A",
+                    name="RACE_HISPANIC",
+                    friendly_name="race hispanic",
+                    type="number",
+                    startIndex=37,
+                    endIndex=38,
+                    required=True,
+                    validators=[validators.validateRace()],
+                ),
+                Field(
+                    item="17B",
+                    name="RACE_AMER_INDIAN",
+                    friendly_name="race aerical indian",
+                    type="number",
+                    startIndex=38,
+                    endIndex=39,
+                    required=True,
+                    validators=[validators.validateRace()],
+                ),
+                Field(
+                    item="17C",
+                    name="RACE_ASIAN",
+                    friendly_name="race asian",
+                    type="number",
+                    startIndex=39,
+                    endIndex=40,
+                    required=True,
+                    validators=[validators.validateRace()],
+                ),
+                Field(
+                    item="17D",
+                    name="RACE_BLACK",
+                    friendly_name="race black",
+                    type="number",
+                    startIndex=40,
+                    endIndex=41,
+                    required=True,
+                    validators=[validators.validateRace()],
+                ),
+                Field(
+                    item="17E",
+                    name="RACE_HAWAIIAN",
+                    friendly_name="race hawaiian",
+                    type="number",
+                    startIndex=41,
+                    endIndex=42,
+                    required=True,
+                    validators=[validators.validateRace()],
+                ),
+                Field(
+                    item="17F",
+                    name="RACE_WHITE",
+                    friendly_name="race white",
+                    type="number",
+                    startIndex=42,
+                    endIndex=43,
+                    required=True,
+                    validators=[validators.validateRace()],
+                ),
+                Field(
+                    item="18",
+                    name="GENDER",
+                    friendly_name="gender",
+                    type="number",
+                    startIndex=43,
+                    endIndex=44,
+                    required=True,
+                    validators=[validators.isInLimits(0, 9)],
+                ),
+                Field(
+                    item="19A",
+                    name="REC_OASDI_INSURANCE",
+                    friendly_name="receives federal disability insurance benefits\
+                         under the social security OASDI program",
```

Comment:
the newline here causes (in my opinion) a spacing issue
<img width="607" alt="image" src="https://github.com/raft-tech/TANF-app/assets/111305129/e26e800c-f85f-4d4a-8359-22f7caba17da">